### ISSUE TITLE
Clone fails on system without LFS

### ISSUE THREAD

**Issue by Lokgic** [nextml/caption-contest-data#7]
Hi,

When I run 

`python unzip_tools.py unzip`

I get

`Archive:  539-responses.csv.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
`

Same result using mac and ubuntu.

When I tried to unzip the files using other means, I would get the zip/cpgz loop on mac, and in ubuntu it would just tell me the file cannot be unzipped. Are the files corrupted or is this a problem on my end? Thanks.

**Comment by stsievert** [comment]
This worked for me:

``` shell
$ git clone https://github.com/nextml/caption-contest-data.git
$ cd caption-contest-data/contests
$ python zip_tools.py unzip
```

Is the problem is with your network connection? What network are you on?

Do you need the ZIP files? We do have summaries of the ZIP files in [contests/summaries][1], and they do a pretty good job of summarizing (plus, they're even available on data.world).

[1]:https://github.com/nextml/caption-contest-data/tree/0d6428771731e040b7d97fc071eff50f5e798c3e/contests/summaries

**Comment by Lokgic** [comment]
Wow that is strange. I was hoping to play around with the responses. That's why I was looking at it. How big are the zip files supposed to be? They all show up as less than 1k for me. I have a pretty good connection. I am stumped.

**Comment by Lokgic** [comment]
I just tried this using windows too. Same thing.

**Comment by stsievert** [comment]
What messages did you see that made you realize you had to install Git LFS?

**Comment by Lokgic** [comment]
Just a hunch, really. I saw that all of the files are properly cloned, except for the larger files. And I noticed the commit log says those files were uploaded via LFS. So I gave it a try.

**Comment by stsievert** [comment]
I’ll try to put in a warning for this in zip_tools.py

**Comment by liamim** [comment]
@stsievert I can take care of this if you want.

**Comment by stsievert** [comment]
I’d appreciate that. I was going to put a check in zip_tools.py. Could you put in a PR for this? Thanks!

On October 23, 2017 at 10:51:02 AM, Liam Marshall (notifications@github.com) wrote:

@stsievert I can take care of this if you want.

**Comment by stsievert** [comment]
Fixed by #8
